commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/SubmissionSource.kt

```diff
@@ -0,0 +1,5 @@
+package ebi.ac.uk.model
+
+enum class SubmissionSource {
```

Comment:
I would suggest: `SubmissionMethod` and for the methods, I would suggest `FILE`, `PAGE_TAB`